state/azure/cosmosdb/cosmosdb.go

```diff
@@ -534,20 +533,6 @@ func populatePartitionMetadata(key string, requestMetadata map[string]string) st
 	return key
 }
 
-func parseTTL(requestMetadata map[string]string) (*int, error) {
```

Comment:
I like this part: having a common method in ttl/utils, instead of repetitive method everywhere.

Reply:
we already had a common method in `metadata/utils.go`, but that one just can't be used in this instance